recipes-core/rauc/files/rauc-mark-good

```diff
@@ -132,12 +160,16 @@ if [ ! -e "$BACKUP_MARKER" ]; then
     mount /dev/niboot/niboot.other "$OTHER_TMP_MOUNT"
     rm -fr "$OTHER_TMP_MOUNT/*"
     cp -R /boot/* "$OTHER_TMP_MOUNT"
-    rm "$OTHER_TMP_MOUNT/slot.raucs"
+    rm -fr "$OTHER_TMP_MOUNT/slot.raucs"
     echo -n > "$BACKUP_MARKER"
     chmod 0444 "$BACKUP_MARKER"
     sync
 fi
 
 # Print message at end so that we have some indication in system log
 # that everything went well
-info "Marked current boot good"
+if ! $USB_BOOT; then
+    info "Marked current boot good"
+else
+    info "Skipped call to rauc mark-good to avoid updating boot sequence"
```

Comment:
Can you make this status message more descriptive, by including information about USB boot? Something like:
```
USB boot detected; skipped RAUC mark-good to avoid updating the boot sequence.
```

Reply:
Done, I like your wording better anyway.